pkg/utils/ocm.go

```diff
@@ -70,3 +70,84 @@ func CreateConnection() *sdk.Connection {
 	}
 	return connection
 }
+
+//This command implements the ocm describe cluster call via osm-sdk.
+//This call requires the ocm API Token https://cloud.redhat.com/openshift/token be available in the OCM_TOKEN env variable.
+//Example: export OCM_TOKEN=$(jq -r .refresh_token ~/.ocm.json)
+func DescribeCluster(clusterID string) (*v1.Cluster, error) {
```

Comment:
To be pedantic, this is the equivalent of `ocm get cluster $clusterID`, rather than `ocm describe cluster $clusterID`. Describe is a GET with some filtering of the data, but this code just does the GET.

Reply:
Removed this function, as I noticed it was already implemented elsewhere.